prime-router/metadata/fhir_mapping/hl7/datatypes/CWE/CodeableConcept.yml

```diff
@@ -0,0 +1,171 @@
+      valueOf: $system
+    version:
+      condition: $version NOT_NULL
+      valueOf: $version
+    extension_cwe-coding:
+      condition: $code NOT_NULL || $display NOT_NULL || $system NOT_NULL || $version NOT_NULL
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: cwe-coding
+        valueString:
+          value: coding
+    extension_cwe-coding-system:
+      condition: $coding-system NOT_NULL
+      generateList: true
+      expressionType: nested
+      vars:
+        coding-system: String, CWE.3
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: cwe-coding-system
+        valueString:
+          value: $coding-system
+
+coding_alt:
+  generateList: true
+  expressionType: nested
+  vars:
+    altCode: String, CWE.4
+    altDisplay: String, CWE.5
+    altSystem: SYSTEM_URL, CWE.6
+    altVersion: String, CWE.8
+  expressionsMap:
+    code:
+      condition: $altCode NOT_NULL
+      valueOf: $altCode
+    display:
+      condition: $altDisplay NOT_NULL
+      valueOf: $altDisplay
+    system:
+      condition: $altSystem NOT_NULL
+      valueOf: $altSystem
+    version:
+      condition: $altVersion NOT_NULL
+      valueOf: $altVersion
+    extension_cwe-coding:
+      condition: $altCode NOT_NULL || $altDisplay NOT_NULL || $altSystem NOT_NULL || $altVersion NOT_NULL
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: cwe-coding
+        valueString:
+          value: alt-coding
+    extension_cwe-coding-system:
+      condition: $coding-system NOT_NULL
+      generateList: true
+      expressionType: nested
+      vars:
+        coding-system: String, CWE.6
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: cwe-coding-system
+        valueString:
+          value: $coding-system
+
+coding_secondary-alt:
+  generateList: true
+  expressionType: nested
+  vars:
+    secondaryAltCode: String, CWE.10
+    secondaryAltDisplay: String, CWE.11
+    secondaryAltSystem: SYSTEM_URL, CWE.12
+    secondaryAltVersion: String, CWE.13
+  expressionsMap:
+    code:
+      condition: $secondaryAltCode NOT_NULL
+      valueOf: $secondaryAltCode
+    display:
+      condition: $secondaryAltDisplay NOT_NULL
+      valueOf: $secondaryAltDisplay
+    system:
+      condition: $secondaryAltSystem NOT_NULL
+      valueOf: $secondaryAltSystem
+    version:
+      condition: $secondaryAltVersion NOT_NULL
+      valueOf: $secondaryAltVersion
+    extension_cwe-coding:
+      condition: $secondaryAltCode NOT_NULL || $secondaryAltDisplay NOT_NULL || $secondaryAltSystem NOT_NULL || $secondaryAltVersion NOT_NULL
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: cwe-coding
+        valueString:
+          value: secondary-alt-coding
```

Comment:
Just as an FYI, this was already a concept that I had to handle for the ELIMS work but did it slightly different
[See](https://github.com/CDCgov/prime-reportstream/blob/12eb307a2fa93414e7d776f3aef8543defbd7867/prime-router/metadata/fhir_mapping/hl7/datatype/CodeableConcept.yml#L50)

I think the name maybe doesn't matter in this case though, I can't think of any implications of changing it